docs/recent_activity/recent_activity.md

```diff
@@ -0,0 +1,60 @@
+---
+description: Monitor recent activity logged actions.
+---
+
+# Recent activity log
+
+## Activity list
+
+On the Back Office, in **Admin** -> **Activity list**, you can consult what actions were taken by users recently.
+
+!!! note
+
+    - To see the Admin menu, you must have the "Setup / Administrate" [permission](permission_system.md).
```

Comment:
This is the label of the permission, not its actual identifier. It could become confusing if I apply that to https://github.com/ibexa/documentation-developer/pull/2161/files#diff-39e3b03985124d185ec800d9ec576a593d8b720c654dba273908abadc2713a63R36

Reply:
It's an option in a drop-down select menu. It's somehow like a button.

```suggestion
    To see the **Admin** menu, you need the **Setup / Administrate** [permission](permission_system.md).
```